ardupilot_ros/usage.md

```diff
@@ -0,0 +1,17 @@
+# Usage Instructions
+
+## Joystick controller
+
+First install the following package
```

Comment:
The README is now out of date. Can you instead just add the docs to the existing README at the top level?